cli/src/commands/compile/run.js

```diff
@@ -1,8 +1,10 @@
+const fs = require('fs');
 const { generateMetadata, generateFile, generateDispatcher } = require('.');
 
 run();
 function run() {
   const metadata = generateMetadata();
+  fs.writeFileSync("./metadata.json", JSON.stringify(metadata, 0, 4));
```

Comment:
What is the reason behind this

Reply:
It was for the testing purposes, I will remove it